packages/frontend/editor-ui/src/components/ParameterInput.vue

```diff
@@ -1591,7 +1592,9 @@ onClickOutside(mapperElRef, onClickOutsideMapper);
 					:title="displayTitle"
 					:placeholder="getPlaceholder()"
 					data-test-id="parameter-input-field"
-					@update:model-value="(valueChanged($event) as undefined) && onUpdateTextInput($event)"
+					@update:model-value="
+						(valueChangedDebounced($event) as undefined) && onUpdateTextInputDebounced($event)
```

Comment:
Looks like `onUpdateTextInput` already calls `valueChanged`, should it only call `onUpdateTextInputDebounced` here?